client/extensions/woocommerce/app/products/product-form-categories-card.js

```diff
@@ -12,13 +12,14 @@ import Card from 'components/card';
 import FormFieldSet from 'components/forms/form-fieldset';
 import FormLabel from 'components/forms/form-label';
 import TokenField from 'components/token-field';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
```

Comment:
Fair enough. I'll not hold up this PR, but I'll probably go through all the files soon.